build/pause/Makefile

```diff
@@ -118,7 +118,13 @@ bin/${BIN.linux}-$(OS)-$(ARCH): $(SRCS)
 			$(TRIPLE)-strip $(foreach binary, $@, ${binary}${EXTENSION})"
 
 bin/wincat-windows-${ARCH}: windows/wincat/wincat.go
-	CGO_ENABLED=0 GOOS=windows GOARCH=${ARCH} go build -o $@ $^
+	mkdir -p bin
+	docker run --rm -u $$(id -u):$$(id -g) -v $$(pwd):/build \
+		--tmpfs /.cache \
+		$(KUBE_CROSS_IMAGE):$(KUBE_CROSS_VERSION) \
+		/bin/bash -c "\
+		cd /build && \
+		CGO_ENABLED=0 GOOS=windows GOARCH=${ARCH} go build -o $@ $^"
```

Comment:
TODO (maybe): plumb through GOTOOLCHAIN so this works even if we haven't bumped kube-cross yet but did bump .go-version.

OTOH: if we start using `toolchain` in the modules / workspace then this is just another place we'll have to remove it, and right now we do update kubecross